test-tools/append-aot-to-wasm/append_aot_to_wasm.py

```diff
@@ -155,7 +158,10 @@ def main(wasm_file: str, aot_file: str, output: str) -> None:
     argparse = argparse.ArgumentParser()
     argparse.add_argument("--wasm", help="a .wasm")
     argparse.add_argument("--aot", help="a .aot")
+    argparse.add_argument(
+        "--ver_str", help="a version string will be used to construct section name"
```

Comment:
How about `--version-string` to show more info? And had better not use `_`, use `-` instead to be consistent with iwasm/wamrc's arguments.